go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/overmindtech/aws-source
 
-go 1.22.5
+go 1.22.4
```

Comment:
Why are downgrading go?

Reply:
one of lifes great mysteries.  :)

go.work in workspace was really angry about being 1.22.4 and this go.mod it 1.22.5

Reply:
I updated the workspace yesterday to use 1.22.5 everywhere because riverqueue pulled us up. please just fetch/rebase everything

Reply:
fixed, not sure why i was missing it. kicked the workspace folder a few times.

Reply:
one of lifes great mysteries.